src/main/java/com/adelinasportfolio/emailservice/controller/EmailController.java

```diff
@@ -0,0 +1,21 @@
+package com.adelinasportfolio.emailservice.controller;
+
+import com.adelinasportfolio.emailservice.model.EmailEntity;
+import com.adelinasportfolio.emailservice.service.EmailService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("api/v1/sendEmail")
```

Comment:
Eu as pune /api/v1 direct in application.properties. Banuiesc ca daca o sa mai ai si alte controlere, tot pe path-ul asta o sa fie. Si as pune, pe controler /email, iar pe methoda /send (ai putea chiar sa nu mai pui nimic pe metoda daca e singurul post - dar banuiesc ca ajuta la readability?). Sa fie clar ca ai controlerul pentru resursa email. Si asa lasi loc si pentru alte metode in controler, daca o sa fie cazul.